code/src/components/PopularList.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const PopularList = ({movies, imgSize, baseURL}) => {
+    return (
+        <div className='movie-list-container'>
+            {movies.map((movie) => (
+                <div 
+                    key={movie.id}
+                    className='movie'
+                    role='link'
+                >    
+                    <Link to={`/movies/${movie.id}`}>
+                        <img 
+                            className='poster-img'
+                            src={`${baseURL}${imgSize}${movie.poster_path}`} 
+                            alt={movie.orginal_title}
+                        />
+                        <div className="overlay">
+                            <div className="overlay-text">
+                                <h2>{movie.original_title}</h2>
+                                <p>{'Released: ' + new Date(movie.release_date).getFullYear()}</p>
+                            </div>
+                        </div>    
+                    </Link>
+                </div>
+            ))}
+        </div>

```

Comment:
Super nice mapping, very clear what's going on. Curious why you put the link inside the div, but it works, so I mean - why not? 😛